doc/api/n-api.md

```diff
@@ -3083,6 +3085,26 @@ method or accessor, then the `this` argument to the callback is the wrapper
 object; the wrapped C++ instance that is the target of the call can be obtained
 then by calling `napi_unwrap()` on the wrapper object.
 
+### *napi_remove_wrap*
+<!-- YAML
+added: REPLACEME
+-->
+```C
+napi_status napi_remove_wrap(napi_env env,
+                             napi_value js_object,
+                             void** result);
+```
+
+ - `[in] env`: The environment that the API is invoked under.
+ - `[in] js_object`: The object associated with the native instance.
+ - `[out] result`: Pointer to the wrapped native instance.
```

Comment:
I guess it's to avoid having to make two API calls. In https://github.com/nodejs/abi-stable-node/issues/266 @sampsongao is trying to do a `napi_unwrap()` followed unconditionally by a `napi_wrap()`. Knowing that an unconditional `napi_wrap()` will follow would allow him to drop in `napi_remove_wrap()` instead of `napi_unwrap()`.